coderedcms/models/wagtailsettings_models.py

```diff
@@ -117,6 +117,12 @@ class Meta:
         related_name='favicon',
         verbose_name=_('Favicon'),
     )
+    navbar_name = models.CharField(
```

Comment:
I don't think this field is necessary here. The `Navbar` object already has a field for the name.

The navbars are a bit confusing in coderedcms. There is one `<nav>` element (which is what all of these settings pertain to). Within that `<nav>` it can render multiple `Navbar` snippets.